Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityControllerConfigurationProfile.asset

```diff
@@ -13,8 +13,16 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   renderMotionControllers: 1
   useDefaultModels: 0
-  overrideLeftHandModel: {fileID: 0}
-  overrideRightHandModel: {fileID: 0}
+  globalLeftHandModel: {fileID: 1670523128530642, guid: 02bef524e73fce64bb75d8e087a9ddb3,
+    type: 2}
+  leftHandModelPoseOffset:
+    position: {x: 0, y: 0, z: 0}
+    rotation: {x: 0, y: 0, z: 0.707, w: 1}
+  globalRightHandModel: {fileID: 1568362058278088, guid: a2749ae6da9ff9c4c84741192bbbefd5,
+    type: 2}
+  rightHandModelPoseOffset:
+    position: {x: 0, y: 0, z: 0}
+    rotation: {x: 0, y: 0, z: -0.707, w: 1}
```

Comment:
model rotation should be in the prefab and not the configuration file. swapping the hand model with a gizmo causes the model to be over-rotated

Reply:
As stated in Chat.  Prefab rotations are only start values, they are overwritten when moved.

Unless you are saying that on Controller initialisation, we capture that value and persist it for the lifetime of the Controller, possibly?